en/micro-integrator/docs/develop/installing-WSO2-Integration-Studio.md

```diff
@@ -25,7 +25,10 @@ Follow the steps given below.
 
 1.  Download the [distribution](https://wso2.com/integration/integration-studio/) for your operating system.
 2.  Extract the downloaded file to get the **IntegrationStudio**
-    application. If you are a MacOS user, be sure to add it to the **Applications** directory.
+    application. 
+    !!! note
+        * If you are a MacOS user, be sure to add it to the **Applications** directory.
+        * If you are a Microsoft Windows user, extract it outside the **Programs** directory.
```

Comment:
SHould we not explain why it should be extracted outside the Programs directory?

Reply:
Fixed.